bindings/java/fdbJNI.cpp

```diff
@@ -1442,14 +1443,13 @@ JNIEXPORT jlong JNICALL Java_com_apple_foundationdb_FDBTransaction_Transaction_1
                                                                                                jint targetBytes,
                                                                                                jint streamingMode,
                                                                                                jint iteration,
-                                                                                               jint matchIndex,
                                                                                                jboolean snapshot,
-                                                                                               jboolean reverse) {
+                                                                                               jboolean reverse,
+                                                                                               jint matchIndex) {
```

Comment:
What happens to 7.1 Java client calls this function? There is no `matchIndex` passed in, I'd assume.

Reply:
good point, see my comment [here](https://github.com/apple/foundationdb/pull/9058#issuecomment-1353781142).